src/bit-u.md

```diff
@@ -513,9 +513,11 @@ a.quote-link:hover{
     <p>Or as we host incredible speakers with Q&A at the monthly ChicagoJS Meetups.</p>
     <a class="button" href="https://www.meetup.com/js-chi/events/"><img src="./static/img/calendar.svg" height="20"> ChicagoJS Meetup Schedule</a>
     <!-- Community calendar temporarily removed until the calendar is consistently updated again. -->
```

Comment:
remove the comment that says its commented out, cause its not commented out anyme. =P

Reply:
Thank you, resolved in next commit.